serverlessworkflow/modules/ROOT/pages/cloud/operator/configuring-knative-eventing-resources.adoc

```diff
@@ -1,60 +1,425 @@
+kind: Trigger
+metadata:
+  name: sonataflow-platform-jobs-service-create-job-trigger
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: job.create
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: sonataflow-platform-jobs-service
+      namespace: example-namespace
+    uri: /v2/jobs/events
+----
+
+.Delete Job events trigger
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: jobs-service-postgresql-delete-job-trigger
+  namespace: example-namespace
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: job.delete
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: sonataflow-platform-jobs-service
+      namespace: example-namespace
+    uri: /v2/jobs/events
+----
+
+== Data Index and Job Service installation
+
+To deploy these services you must use a `SonataFlowPlatform CR` and configure it according to the xref:cloud/operator/supporting-services.adoc[Supporting Services guide].
+Finally, prior to deployment into the cluster, you must add the `env` variable shown below to the field `spec.jobService.podTemplate.container`.
+
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+  namespace: example-namespace
+spec:
+  services:
+    dataIndex:
+      # Data Index requires no additional configurations to use knative eventing.
+      # Use the configuration of your choice according to the Supporting Services guide.
+    jobService:
+      podTemplate:
+        container:
+          env:
+            - name: MP_MESSAGING_OUTGOING_KOGITO_JOB_SERVICE_JOB_STATUS_EVENTS_HTTP_URL <1>
+              value: http://broker-ingress.knative-eventing.svc.cluster.local/example-namespace/default <2>
+----
+
+<1> Fixed env variable name that contains the URL of the Broker created in <<_configuring_the_knative_broker>>.
+<2> To query the Broker URL <<querying_broker_url, see>>.
+
+== Workflow configuration
+
+=== SonataFlow CR configuration
+
+To configure a workflow you must create a `SonataFlow CR` that fulfills your requirements as usual.
+And finally, prior to deployment into the cluster, add the `env` variables shown below to the field `spec.podTemplate.container`.
+
+.Workflow configuration
+[source,yaml]
+----
 apiVersion: sonataflow.org/v1alpha08
 kind: SonataFlow
 metadata:
-...
+  name: example-workflow
+  namespace: example-namespace
+  annotations:
+    sonataflow.org/description: Example Workflow that show Knative Eventing configuration.
+    sonataflow.org/version: 0.0.1
+    sonataflow.org/profile: preview
 spec:
-  sink:
-    ref: <1>
-      name: default
-      namespace: greeting
-      apiVersion: eventing.knative.dev/v1
-      kind: Broker
+  podTemplate:
+    container:
+      env:
+        - name: K_SINK <1>
+          value: http://broker-ingress.knative-eventing.svc.cluster.local/example-namespace/default <2>
```

Comment:
So this is the workaround? Instead of creating a SinkBinding because of the volume issue?

Reply:
yep!